helpers/install_bigartm.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Please remove this. We have `ast2vec bigartm` and the dependencies should always be installed individually because people have different OSes

Reply:
yes, but can I leave it with the comment that it works on ubuntu and mac?
`ast2vec bigartm` does not install python package and I need it.

Reply:
or we can add option for python package installation to  `ast2vec bigartm`.

Reply:
We should install it unconditionally

Reply:
why? it is an option to install package.
here https://github.com/src-d/ast2vec/blob/master/topic_modeling.md we need only executable. 
And we need a package only in snippet_ranger case.

Reply:
The less options, the better. We are already running python so let's install the package unconditionally

Reply:
ok.